src/view/App.tsx

```diff
@@ -51,15 +52,11 @@ class App extends React.Component<any, any> {
         let cronDate = new Date(lastCron);
         let now = new Date();
         if (cronDate.getDate() != now.getDate() || (cronDate.getMonth() != now.getMonth() || cronDate.getFullYear() != now.getFullYear())) {
-            // return (
```

Comment:
Can you please not remove the existing cron implementation? We fixed it in commit d10a2ca747c1ada5e255bef6d39eddc45b0dcf26 Atleast provide a reason for it.

Reply:
ok，got it.